README.md

```diff
@@ -326,12 +324,12 @@ Note: it is also possible to generate a **source** folder containing added/chang
 The simplest option to deploy the incremental changes is to use `force:source:deploy` command with `-x` parameter:
 
 ```sh
-sfdx force:source:deploy -x package/package.xml --postdestructivechanges destructiveChanges/destructiveChanges.xml
+sf project deploy start -x package/package.xml --post-destructive-changes destructiveChanges/destructiveChanges.xml
 ```
 
 And voilà! 🥳
 
-However, keep in mind that the above command will fail if the destructive change was supposed to be executed before the deployment (i.e. as `--predestructivechanges`), or if a warning occurs during deployment. Make sure to protect your CI/CD pipeline from those scenarios, so that it doesn't get stuck by a failed destructive change.
+However, keep in mind that the above command will fail if the destructive change was supposed to be executed before the deployment (i.e. as `--pre-destructive-changes=`), or if a warning occurs during deployment. Make sure to protect your CI/CD pipeline from those scenarios, so that it doesn't get stuck by a failed destructive change.
```

Comment:
```suggestion
However, keep in mind that the above command will fail if the destructive change was supposed to be executed before the deployment (i.e. as `--pre-destructive-changes`), or if a warning occurs during deployment. Make sure to protect your CI/CD pipeline from those scenarios, so that it doesn't get stuck by a failed destructive change.
```